string cmdType
float32 dist_or_deg
---
bool isComplete
string errorMsg
